src/wallet-instance-attestation/issuing.ts

```diff
@@ -1,76 +1,86 @@
-import {
-  type CryptoContext,
-  decode as decodeJwt,
-} from "@pagopa/io-react-native-jwt";
-import { verify as verifyJwt } from "@pagopa/io-react-native-jwt";
+import { type CryptoContext } from "@pagopa/io-react-native-jwt";
 import { SignJWT, thumbprint } from "@pagopa/io-react-native-jwt";
 import { JWK, fixBase64EncodingOnKey } from "../utils/jwk";
-import { WalletInstanceAttestationRequestJwt } from "./types";
-import uuid from "react-native-uuid";
-import { WalletInstanceAttestationIssuingError } from "../utils/errors";
-import type { WalletProviderEntityConfiguration } from "../trust/types";
+import { getWalletProviderClient } from "../client";
+import type { IntegrityContext } from "..";
+import { z } from "zod";
 
-async function getAttestationRequest(
+export async function getAttestationRequest(
+  challenge: string,
   wiaCryptoContext: CryptoContext,
-  walletProviderEntityConfiguration: WalletProviderEntityConfiguration
+  integrityContext: IntegrityContext,
+  walletProviderBaseUrl: string
 ): Promise<string> {
   const jwk = await wiaCryptoContext.getPublicKey();
   const parsedJwk = JWK.parse(jwk);
   const keyThumbprint = await thumbprint(parsedJwk);
   const publicKey = { ...parsedJwk, kid: keyThumbprint };
 
+  const clientData = {
+    challenge,
+    // jwk
+  };
```

Comment:
```suggestion
  const clientData = {
    challenge,
    jwk_thumbprint: keyThumbprint
  };
```

Reply:
Done in [611a422](https://github.com/pagopa/io-react-native-wallet/pull/86/commits/611a42293f8a53f2b2f86e6c93a4dd79a108817a), thanks.